core/includes/updates/230.php

```diff
@@ -66,6 +66,16 @@ public function run(): void
         Settings::set('discord_widget_theme', $discord_widget_theme, 'Discord Integration');
         $this->_cache->eraseAll();
 
+        // Add admincp.core.backups permission to Admin group
+        $admin_group = Group::find(1);
+        $permissions = json_decode($admin_group->permissions, true);
+        if (!isset($permissions['admincp.core.backups'])) {
+            $permissions['admincp.core.backups'] = 1;
+            DB::getInstance()->update('groups', $admin_group->id, [
+                'permissions' => json_encode($permissions),
```

Comment:
All of this can still be removed, We have not updated admin group perms for ages as it already had Adminstrator perm